entity-framework/core/get-started/index.md

```diff
@@ -1,33 +1,164 @@
-  * [New Database](full-dotnet/new-db.md)
-  * [Existing Database](full-dotnet/existing-db.md)
+Install the following software:
 
-> [!NOTE]  
-> These tutorials and the accompanying samples have been updated to use EF Core 2.1. However, in the majority of cases it should be possible to create applications that use previous releases, with minimal modification to the instructions. 
+# [.NET Core CLI](#tab/netcore-cli)
+
+* [.NET Core 3.0 SDK](https://www.microsoft.com/net/download/core).
+
+# [Visual Studio](#tab/visual-studio)
+
+* [Visual Studio 2019 version 16.3 or later](https://www.visualstudio.com/downloads/) with this  workload:
+  * **.NET Core cross-platform development** (under **Other Toolsets**)
+
+---
+
+## Create a new project
+
+# [.NET Core CLI](#tab/netcore-cli)
+
+``` Console
+dotnet new console -o EFGetStarted
+cd EFGetStarted
+```
+
+# [Visual Studio](#tab/visual-studio)
+
+* Open Visual Studio
+* Click **Create a new project**
+* Select **Console App (.NET Core)** with the **C#** tag and click **Next**
+* Enter **EFGetStarted** for the name and click **Create**
+
+---
+
+## Install Entity Framework Core
+
+To install EF Core, you install the package for the EF Core database provider(s) you want to target. This tutorial uses SQLite because it runs on all platforms that .NET Core supports. For a list of available providers, see [Database Providers](../providers/index.md).
+
+# [.NET Core CLI](#tab/netcore-cli)
+
+``` Console
+dotnet add package Microsoft.EntityFrameworkCore.Sqlite
+```
+
+# [Visual Studio](#tab/visual-studio)
+
+* **Tools > NuGet Package Manager > Package Manager Console**
+* Run the following commands:
+
+  ``` PowerShell
+  Install-Package Microsoft.EntityFrameworkCore.Sqlite
+  ```
+
+Tip: You can also install packages by right-clicking on the project and selecting **Manage NuGet Packages**
+
+---
+
+## Create the model
+
+Define a context class and entity classes that make up the model.
+
+# [.NET Core CLI](#tab/netcore-cli)
+
+* In the **Models** folder create **Model.cs** with the following code
+
+# [Visual Studio](#tab/visual-studio)
+
+* Right-click on the project and select **Add > Class**
+* Enter **Model.cs** as the name and click **Add**
+* Replace the contents of the file with the following code
+
+---
+
+[!code-csharp[Main](../../../samples/core/GetStarted/Model.cs)]
+
+EF Core can also [reverse engineer](../managing-schemas/scaffolding.md) a model from an existing database.
+
+Tip: In a real app, you put each class in a separate file and put the [connection string](../miscellaneous/connection-strings.md) in a configuration file or environment variable. To keep the tutorial simple, everything is contained in one file.
+
+## Create the database
+
+The following steps use [migrations](xref:core/managing-schemas/migrations/index) to create a database.
+
+# [.NET Core CLI](#tab/netcore-cli)
+
+* Run the following commands:
+
+  ``` Console
+  dotnet tool install --global dotnet-ef --version 3.0.0-*
```

Comment:
Will it match 3.0.0-* to 3.0.0 once we RTM?

Reply:
Will update on the day we publish as part of #1588